modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -132,6 +133,25 @@ def log_special_issues
       Rails.logger.info("ClaimsApi: Claim[#{id}] contains the following special issues - #{special_issues}")
     end
 
+    def validate_service_dates
+      service_periods = form_data.dig('serviceInformation', 'servicePeriods')
+
+      service_periods.each do |service_period|
+        start_date = if service_period['activeDutyBeginDate'].present?
+                       Date.parse(service_period['activeDutyBeginDate'])
+                     end
+        end_date = (Date.parse(service_period['activeDutyEndDate']) if service_period['activeDutyEndDate'].present?)
+
+        if start_date.present? && end_date.blank?
+          next
+        elsif start_date.blank?
+          errors.add :activeDutyBeginDate, 'must be present'
```

Comment:
@SlakrHakr don't some of the required validations occur elsewhere? I thought that the `required` fields aspect gets enforced by JSON schema

Reply:
probably, i was just trying to be as thorough as possible.
also, whos to say sometime in the future this record is edited outside of the workflow we expect today, not the end of the world to come all bases in a couple different places i dont think